pkg/resource/alias/hooks.go

```diff
@@ -162,40 +208,11 @@ func (rm *resourceManager) getFunctionEventInvokeConfig(
 			Qualifier:    ko.Spec.Name,
 		},
 	)
-
 	rm.metrics.RecordAPICall("GET", "GetFunctionEventInvokeConfig", err)
 
+	err = rm.setFunctionEventInvokeConfigFromResponse(ko, getFunctionEventInvokeConfigOutput, err)
```

Comment:
instead of passing this err as an argument can you please handle this case before? like a if err != nil ...

Reply:
Done